inlineplz/linters/__init__.py

```diff
@@ -252,7 +255,18 @@ def installed(config):
         return False
 
 
-def lint(install=False, autorun=False, ignore_paths=None):
+def run_config(config, config_dir):
+    if dotfiles_exist(config) and config.get('run'):
+        return config.get('run')
+    if not (config_dir and dotfiles_exist(config, config_dir)):
+        config_dir = os.path.abspath(os.path.join(HERE, 'config'))
+    return [
+        os.path.normpath(item.format(config_dir=config_dir))
+        for item in (config.get('rundefault') or config.get('run'))
+        ]
```

Comment:
`prospector: pep8: closing bracket does not match indentation of opening bracket's line (E123)`